electrum/plugins/hw_wallet/plugin.py

```diff
@@ -379,3 +361,7 @@ def text_ignore_old_fw_and_continue(self) -> str:
             return str(self) + "\n\n" + suffix
         else:
             return suffix
+
+
+class OperationCancelled(UserFacingException):
```

Comment:
How does this differ from `UserCancelled`? If we need both, maybe add a comment here.